src/sentry/static/sentry/app/views/performance/landing/content.tsx

```diff
@@ -114,45 +122,69 @@ class LandingContent extends React.Component<Props, State> {
     switch (display) {
       case LandingDisplayField.ALL:
         return this.renderLandingAll(summaryConditions);
-      case LandingDisplayField.FRONTEND:
-        return this.renderLandingFrontend(summaryConditions);
+      case LandingDisplayField.FRONTEND_PAGELOAD:
+        return this.renderLandingFrontend(summaryConditions, true);
+      case LandingDisplayField.FRONTEND_NAVIGATION:
+        return this.renderLandingFrontend(summaryConditions, false);
       case LandingDisplayField.BACKEND:
         return this.renderLandingBackend(summaryConditions);
       default:
         throw new Error(`Unknown display: ${display}`);
     }
   }
 
-  renderLandingFrontend = summaryConditions => {
+  renderLandingFrontend = (summaryConditions, isPageload) => {
     const {organization, location, projects, eventView, setError} = this.props;
 
-    const axisOptions = getFrontendAxisOptions(organization);
+    const op = isPageload ? 'pageload' : 'navigation';
+
+    const frontendView = eventView.clone();
+    const conditions = tokenizeSearch(frontendView.query);
+    const newConditions = new QueryResults([]); // Add conditions just to include in summary as transaction.op you want to likely keep
+    conditions.setTagValues('transaction.op', [op]);
+    newConditions.setTagValues('transaction.op', [op]);
```

Comment:
Should these be added to the top level event view rather than here?

Reply:
No, since I don't want it to appear in the searchbar, so we'd need to split it anyway, this can be fixed once I do the work to allow 'hidden' conditions on eventview.